src/locale/locales/ru/messages.po

```diff
@@ -0,0 +1,6232 @@
+#: src/view/com/posts/FollowingEmptyState.tsx:57
+#: src/view/com/posts/FollowingEndOfFeed.tsx:58
+msgid "Find accounts to follow"
+msgstr "Найдите учетные записи для подписки"
+
+#: src/view/screens/Search/Search.tsx:469
+msgid "Find posts and users on Bluesky"
+msgstr "Найти посты и пользователей в Bluesky"
+
+#: src/view/screens/PreferencesFollowingFeed.tsx:110
+msgid "Fine-tune the content you see on your Following feed."
+msgstr "Выберите, что вы хотите видеть в своей ленте подписок."
+
+#: src/view/screens/PreferencesThreads.tsx:60
+msgid "Fine-tune the discussion threads."
+msgstr "Настройте отображение обсуждений."
+
+#: src/screens/Onboarding/index.tsx:35
+msgid "Fitness"
+msgstr "Фитнес"
+
+#: src/screens/Onboarding/StepFinished.tsx:164
+msgid "Flexible"
+msgstr "Гибкий"
+
+#: src/view/com/modals/EditImage.tsx:116
+msgid "Flip horizontal"
+msgstr "Отзеркалить горизонтально"
+
+#: src/view/com/modals/EditImage.tsx:121
+#: src/view/com/modals/EditImage.tsx:288
+msgid "Flip vertically"
+msgstr "Отзеркалить вертикально"
+
+#: src/components/ProfileHoverCard/index.web.tsx:412
+#: src/components/ProfileHoverCard/index.web.tsx:423
+#: src/screens/Profile/Header/ProfileHeaderStandard.tsx:244
+#: src/view/com/post-thread/PostThreadFollowBtn.tsx:146
+#: src/view/com/profile/ProfileHeaderSuggestedFollows.tsx:247
+msgid "Follow"
+msgstr "Подписаться"
+
+#: src/view/com/profile/FollowButton.tsx:69
+msgctxt "action"
+msgid "Follow"
+msgstr "Подписаться"
+
+#: src/screens/Profile/Header/ProfileHeaderStandard.tsx:230
+#: src/view/com/post-thread/PostThreadFollowBtn.tsx:128
+msgid "Follow {0}"
+msgstr "Подписаться на {0}"
+
+#: src/view/com/posts/AviFollowButton.tsx:71
+msgid "Follow {name}"
+msgstr "Подписаться на {name}"
+
+#: src/view/com/profile/ProfileMenu.tsx:244
+#: src/view/com/profile/ProfileMenu.tsx:255
+msgid "Follow Account"
+msgstr "Подписаться на учетную запись"
+
+#: src/view/com/post-thread/PostThreadFollowBtn.tsx:144
+msgid "Follow Back"
+msgstr "Подписаться в ответ"
+
+#: src/view/com/profile/ProfileCard.tsx:227
+msgid "Followed by {0}"
+msgstr "Подписаны {0}"
+
+#: src/view/com/modals/Threadgate.tsx:99
+msgid "Followed users"
+msgstr "Ваши подписки"
+
+#: src/view/screens/PreferencesFollowingFeed.tsx:153
+msgid "Followed users only"
+msgstr "Только ваши подписки"
+
+#: src/view/com/notifications/FeedItem.tsx:173
+msgid "followed you"
+msgstr "подписал(ся/ись) на вас"
+
+#: src/view/com/profile/ProfileFollowers.tsx:104
+#: src/view/screens/ProfileFollowers.tsx:25
+msgid "Followers"
+msgstr "Подписчики"
+
+#: src/components/ProfileHoverCard/index.web.tsx:411
+#: src/components/ProfileHoverCard/index.web.tsx:422
+#: src/screens/Profile/Header/ProfileHeaderStandard.tsx:242
+#: src/view/com/post-thread/PostThreadFollowBtn.tsx:149
+#: src/view/com/profile/ProfileFollows.tsx:104
+#: src/view/screens/Feeds.tsx:683
+#: src/view/screens/ProfileFollows.tsx:25
+#: src/view/screens/SavedFeeds.tsx:415
+msgid "Following"
+msgstr "Подписан"
```

Comment:
It looks a little strange and unusual.
It might be worth replacing it with “Подписаны” or “В подписках”
![изображение](https://github.com/bluesky-social/social-app/assets/59219907/40f8489a-806f-47da-9048-f8deabc2258a)

Reply:
Changed to "В подписках"